cmd/client/capture.go

```diff
@@ -99,11 +99,11 @@ var CaptureCreate = &cobra.Command{
 		capture.ReassembleTCP = reassembleTCP
 		capture.LayerKeyMode = layerKeyMode
 		capture.RawPacketLimit = rawPacketLimit
-		capture.ExtraLayers = layers
+		capture.ExtraLayers = int(layers)
```

Comment:
why changing this to int ? later it will be converted again the flow.ExtraLayers

Reply:
I changed the type of ExtraLayers to `int` to avoid having api/types/types.go depend on the `flow` package